frontend/simple/views/containers/contributions/IncomeForm.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <modal ref="modal" :isActive="true" @close="cancelForm">
+    <form class="modal-card-content c-wrapper" @submit.prevent="verifyForm" novalidate="true">
+      <i18n tag="h2" class="title is-3">Income Details</i18n>
+      <div class="columns is-mobile c-wrapper-columns">
+        <div class="column c-form">
+          <fieldset class="fieldset">
+            <i18n tag="legend" class="sr-only">Income details</i18n>
+
+            <div class="field is-narrow gi-fieldGroup">
+              <i18n tag="p" class="label">Do you make at least {{fakeStore.mincomeFormatted}} per month?</i18n>
+              <i18n tag="span" class="help has-text-danger gi-help" v-if="$v.form.option.$error">{{infoRequired}}</i18n>
+              <div class="control">
+                <label class="gi-tick">
+                  <input class="gi-tick-input" type="radio" value="no" v-model="form.option" @change="resetFormVerify">
+                  <span class="gi-tick-custom"></span>
+                  <i18n>No, I don't</i18n>
+                </label>
+                <label class="gi-tick">
+                  <input class="gi-tick-input" type="radio" value="yes" v-model="form.option" @change="resetFormVerify">
+                  <span class="gi-tick-custom"></span>
+                  <i18n>Yes, I do</i18n>
+                </label>
+              </div>
+            </div>
+
+            <input-amount v-if="needsIncome" key="income"
+              label="What's your monthly income?"
+              :error="inputIncomeError"
+              @input="verifyInputIncome"
+              :value="form.income"
+              :placeholder="L('amout')"
+            >
+              <template slot="help">
+                <text-who :who="['Rick', 'Carl', 'Kim']"></text-who>
+                <i18n>will ensure you meet the mincome</i18n>
+              </template>
+            </input-amount>
+
+            <input-amount v-else-if="canPledge" key="pledge"
```

Comment:
What is this `key="pledge"` doing? (And similar question for line 27 above)

Reply:
I'm gonna have a closer look at this, but two things jump out at me:

1. It's `:key=` not `key=` (which does nothing)
2. Where is the corresponding `v-for`?

Reply:
hum... strange... I'll check back again the code to reproduce the bug and show you

Reply:
oh now it doesn't happen the bug... It happened when it was a simple input element instead of a component (as it is now), maybe that's why... I'll remove the `key` attrs then